src/lib.rs

```diff
@@ -17,7 +17,23 @@ use std::{
 };
 use tokio_02::runtime::{Handle, Runtime};
 
+mod io;
+pub use self::io::IoCompat;
```

Comment:
ext trait?

Reply:
We would need a lot of ext traits to do that.